src/wp-content/themes/twentytwentytwo/inc/block-patterns.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Twenty Twenty-Two: Block Patterns
+ *
+ * @since 1.0
+ */
+
+if ( ! function_exists( 'twentytwentytwo_register_block_patterns' ) ) :
+	function twentytwentytwo_register_block_patterns() {
+		if ( function_exists( 'register_block_pattern_category' ) ) {
+			register_block_pattern_category(
+				'twentytwentytwo-general',
+				array( 'label' => __( 'Twenty Twenty-Two General', 'twentytwentytwo' ) )
+			);
+			register_block_pattern_category(
+				'twentytwentytwo-footers',
+				array( 'label' => __( 'Twenty Twenty-Two Footers', 'twentytwentytwo' ) )
+			);
+			register_block_pattern_category(
+				'twentytwentytwo-headers',
+				array( 'label' => __( 'Twenty Twenty-Two Headers', 'twentytwentytwo' ) )
+			);
+			register_block_pattern_category(
+				'twentytwentytwo-query',
+				array( 'label' => __( 'Twenty Twenty-Two Posts', 'twentytwentytwo' ) )
+			);
+			register_block_pattern_category(
+				'twentytwentytwo-pages',
+				array( 'label' => __( 'Twenty Twenty-Two Pages', 'twentytwentytwo' ) )
+			);
+		}
+		if ( function_exists( 'register_block_pattern' ) ) {
```

Comment:
The theme states that it requires at least WP `5.8`. `register_block_pattern` and `register_block_pattern_category` were introduced in WP 5.5.

I know this is defensive coding, but since WordPress will no longer activate a theme that requires a newer version of WordPress (since [5.5 coincidentally](https://core.trac.wordpress.org/ticket/48491)), I'm wondering if we could remove these checks.

Looking back, the Gutenberg plugin has required WP 5.6 since version 10.2. With that in mind, I think it's safe to say that any site installing a FSE theme with the Gutenberg plugin active will be on WP 5.6 or higher.